objectstore-service/src/lib.rs

```diff
@@ -18,17 +20,21 @@ use crate::backend::{BackendStream, BoxedBackend};
 pub use backend::BigTableConfig;
 pub use metadata::*;
 
+/// The threshold up until which we will go to the small backend.
+const SMALL_THRESHOLD: usize = 50 * 1024; // 50 KiB
```

Comment:
Let's set this at 1MB right away, all backends we're evaluating are efficient up to that size, and this covers our use cases well.